README.md

```diff
@@ -0,0 +1,5 @@
+# Discussion: UnixFS Specification
+
+This repo is used to discuss and draft a specification for the next version of UnixFS (the DAG node format used to represent *files and directories* on the IPFS network). You can find and participate in discussion about desired features in [the issues](https://github.com/ipfs/ipld-unixfs/issues) and an [initial specification draft in PR #2](https://github.com/ipfs/ipld-unixfs/pull/2).
+
+When the specification draft is complete, it will be moved to the `ipfs/specs` repo at https://github.com/ipfs/specs/tree/master/unixfs and this repo will likely be re-dedicated to building the **Go** implementation of UnixFS.
```

Comment:
Which part, adding it to the `specs` repo or repurposing this one?

When I talked with @whyrusleeping on Slack, it sounded like he was saying both were pretty certain (also that it might make sense to rename this repo to `go-*`, but that was not at all certain). @whyrusleeping, did I misunderstand?

Reply:
I was not part of this discussion.  @whyrusleeping specifically asked me to create this repo for the point of moving the spec along.  He never mentioned anything about moving it to `ipfs/specs` to me.

Reply:
Ok. It sounds like there are some parts of this that people have different understandings of. This seems like a good opportunity to get that cleared up.

So…

- Is the intent to copy this back to `ipfs/specs/unixfs` when it’s ready correct/agreed on?
- What does everyone understand to be the future of this repo after a draft for the next version of the UnixFS spec is done? (If this stays separate from `ipfs/specs`, what does that mean for future work on other specs that are currently housed there?)

Reply:
Okay.